content/courses/program-optimization/lookup-tables.md

```diff
@@ -376,137 +360,149 @@ const transactionSignature = await connection.sendTransaction(transaction);
 
 Let's go ahead and practice using lookup tables!
 
-this lab will guide you through the steps of creating, extending, and then using
-a lookup table in a versioned transaction.
+This lab will guide you through creating, extending, and using a lookup table in
+a versioned transaction.
 
 #### 1. Get the starter code
 
-To begin, download the starter code from the starter branch of this
+To begin, download the starter code from the "starter" branch of this
```

Comment:
You can probably do without this repo. `initializeKeypair` is in `helpers`. Just ask people to make a file. See 'don't make custom helper functions' in https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#jsts